src/app/issure/advance-editor/advance-editor.component.ts

```diff
@@ -521,12 +533,22 @@ export class AdvanceEditorComponent implements OnInit {
       tempFjson[fieldObj.key]['description'] = fieldObj.description
     }
 
+    if(fieldObj.hasOwnProperty('validate') && fieldObj.validate.minLength){
```

Comment:
@suvarnakale same code repeated here, create one fuction and write login that.
checkValidate(tempjson, element, validateType) .
validateType will be **max, min or customMessage**

Reply:
okay